app/models/package_manager/maven.rb

```diff
@@ -72,7 +72,7 @@ def self.project(name)
 
     def self.mapping(project)
       base_url = "http://repo1.maven.org/maven2/#{project[:groupId]}/#{project[:artifactId]}"
-      latest_version = project[:versions].last[:number]
+      latest_version = project[:versions].sort_by {|version| Date.parse(version[:published_at])}.reverse.first[:number]
```

Comment:
I'm wondering if Date.parse could raise an error ever, and what would happen in that case. But I guess we'd see the exception and could clean it up once we know the reason it raises and if it ever does.